### PR TITLE
Formatting dao proposal's card

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Formatting dao proposals' cards.
Screenshot from 2023-12-03 15-44-43

### Changes

- Made titles bold
- Changed view details link to a tooltip
- Added some styling to the card
### Related Issues

#1502
### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/dashboard/dao_view.vue [hunk 1]
```diff
@@ -36,28 +36,32 @@
...
-            <v-card-title class="pa-0 mb-5" v-if="proposal.action">
+            <v-card-title class="pa-0 mb-5 font-weight-bold" v-if="proposal.action">
               {{ proposal.action }}
             </v-card-title>
-            <!-- <v-card-subtitle class="pb-0">
-              <p class="font-weight-bold">
-                Proposal hash: <span class="text--secondary">{{ proposal.hash }}</span>
-              </p>
-            </v-card-subtitle> -->
+            <v-divider class="mt-1 mb-5 text-red-700" v-if="proposal.action" />
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/dao_view.vue:42]: might be better to just wrap these two components in a div with a single v-if="proposal.action"

**Comment by amiraabouhadid** [comment]
The details button should always be on the right even when there is no remark. Looks odd here, no?
image

**Review by zaelgohary — CHANGES_REQUESTED** [review]
Whenever I click on details, it redirects to the playground in a different tab as seen below.

[details_btn.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/6bdb3344-616d-4fb7-9670-dd35fd9165c2)

**Comment by maayarosama** [comment]
> Whenever I click on details, it redirects to the playground in a different tab as seen below.
> 
>  details_btn.webm

As all proposals on devnet consists of summy data, the details button won't redirect to anything. However if you test against mainnet you can see that it redirects to the right link.

**Comment by zaelgohary** [comment]
> > Whenever I click on details, it redirects to the playground in a different tab as seen below.
> > details_btn.webm
> 
> As all proposals on devnet consists of summy data, the details button won't redirect to anything. However if you test against mainnet you can see that it redirects to the right link.

Then the button should only be visible in mainnet only, no?
فا

**Comment by maayarosama** [comment]
> > > Whenever I click on details, it redirects to the playground in a different tab as seen below.
> > > details_btn.webm
> > 
> > 
> > As all proposals on devnet consists of summy data, the details button won't redirect to anything. However if you test against mainnet you can see that it redirects to the right link.
> 
> Then the button should only be visible in mainnet only, no?

A per discussion with @AhmedHanafy725 it should be visible in all nets, The user who creates the proposal should always add accurate data.

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**